src/app/add-content/add-content.component.scss

```diff
@@ -0,0 +1,146 @@
+#content-page {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  width: 100%;
+  height: 100%;
+}
+
+#content {
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  height: 100%;
+}
+
+#content > form, div {
+  width: 50%;
+
+}
+
+#content-preview {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: flex-start;
+  height: 40vw;
+  img, video {
+    width: 60%;
+    margin-top: 6%;
+    height: fit-content;
+    border-radius: 0.75vw;
+  }
+  p {
+    font-size: 150%;
+  }
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-evenly;
+  height: fit-content;
+  gap: 1.5vw;
+}
+
+.form-line {
+  align-self: center;
+  background-color: #99B8A9;
+  padding: 1vw 2vw 1vw 2vw;
+  border-radius: 0.75vw;
+  display: flex;
+  width: 75%;
+  font-size: 150%;
+}
+
+#submit-line {
+  align-self: center;
+  width: 75%;
+  margin-left: 8%;
+  display: flex;
+  justify-content: end;
+}
+
+input[type=submit] {
+  background-color: #1dbf70;
+  padding: 1vw 3vw 1vw 3vw;
+  border: none;
+  border-radius: 0.75vw;
+  font-size: 180%;
+  color: black;
+  transition: 0.2s ease-in-out;
+  font-weight: bolder;
+}
+
+input[type=submit]:hover:not(:disabled) {
```

Comment:
il n'y a pas moyen de factoriser un peu avec le SCSS ?

Reply:
j'ai remplacé le :not(:disabled) par un :enabled

Reply:
je parle aussi d'imbriquer les input du genre 
`input {
    blablabla
    ...
    :hover {
       blablabla
    }
    ....
}`

Reply:
j'ai essayé de faire comme ça, ça prends pas en compte les choses dans le :hover nesté

Reply:
my bad, j'ai trouvé, il manquait le & avant le :hover nesté